README.md

```diff
@@ -11,21 +11,21 @@ This adds compatibility for [TiDB](https://github.com/pingcap/tidb) to Django.
 
 ### Prerequisites
 
-Before installing django-tidb, ensure you have a MySQL driver installed. You can choose either `mysqlclient`(recommended) or `pymysql`.
```

Comment:
Is there any known issues with PyMySQL? Or else this seems confusing.

Reply:
`pymysql` is not officially supported by `django`, it's own [django-integrate-test](https://github.com/PyMySQL/PyMySQL/actions/runs/5425015023/job/14692541392) also failed now

Reply:
get